server/middleware/error.js

```diff
@@ -6,11 +6,25 @@ exports.notFound = (req, res, next) => {
 
 exports.errorHandler = (err, req, res, next) => {
   //  set locals, only providing error in development
+
   res.locals.message = err.message;
   res.locals.error = req.app.get("env") === "development" ? err : {};
 
-  const statusCode = res.statusCode === 200 ? 500 : res.statusCode;
 
-  res.status(statusCode);
+  let statusCode = res.statusCode === 200 ? 500 : res.statusCode;
```

Comment:
If the statusCode is equal to 200 we say make it 500??

Reply:
This error handler is becoming messy, I suggest we just set the status in the controllers before we send any response back?